### PR TITLE
readme: improve important content visibility

### PR DESCRIPTION
**Pull request by MarcoWorms**
Remove and change some words to make the Readme text more concise.

I hope this helps making the Readme easier to read :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,20 +2,21 @@
...
 
-- Your logs will have a unique id.
-- Request and responses will have the same id. You can pass this id to another service, making possible to track the path of a request across your services.
-- It's possible to hide secret information based on regex.
-- Escriba use JSON format and adds extra information to your logs as pid, hostname, level, startTime, latency.
-- You can tell to Escriba to log only some props from a request or response.
-- If you have some routes that aren't important you can add rules to skip this routes. Also it's possible to skip a route based on methods or just skip body property from a route.
+- JSON format
+- Unique id
+- Request and responses share the same This enables tracking the path of a request across your services
```
Comment (mccraveiro) [inline README.md]: > share the same This 
?

Comment (MarcoWorms) [inline README.md]: oops

**Comment by vitorabner** [comment]
Thanks!

File: README.md [hunk 2]
```diff
@@ -2,20 +2,21 @@
...
-- Escriba use JSON format and adds extra information to your logs as pid, hostname, level, startTime, latency.
-- You can tell to Escriba to log only some props from a request or response.
-- If you have some routes that aren't important you can add rules to skip this routes. Also it's possible to skip a route based on methods or just skip body property from a route.
+- JSON format
+- Unique id
+- Request and responses share the same id. This enables tracking the path of a request across your services
+- Hide secret information based on regex
+- Adds extra information to your logs as `pid`, `hostname`, `level`, `startTime`, and `latency`
+- Filter props from request and/or response
+- Skip this routes based on methods/rules/body props
```
Comment (rctoliveira) [inline README.md]: "Skip these routes" or "Skip this route"

File: README.md [hunk 3]
```diff
@@ -53,9 +56,11 @@ logger.info({ text: 'Setting user permission', password: 'abc' }, { id: cuid(),
 
 ## Http logger
 
-To log http request and response use the httpLogger. For this example we'll long only some properties: id, body and statusCode. 
+Use `httpLogger` to log an http request and response.
+
+For this example we'll long only some properties: `id`, `body` and `statusCode`. 
 
-Also we'll skip status route, options method and body property from routes that ends with .csv or .xlsx.
+We'll also skip status route, options method and body property from routes that ends with .csv or .xlsx.
```
Comment (rctoliveira) [inline README.md]: "from routes that end" or "from route that ends"

File: README.md [hunk 4]
```diff
@@ -96,19 +101,19 @@ app.get('/room/:id', roomController.index)
...
+This id is injected in the `req` object, so if you need to log some extra information between a request and response just do something like this:
 
 ```js
 logger.info('some controller information', { id: req.id })
 ```
 
 ## Examples
 
-The log-generator inside examples folder will run a nodejs application that will make request for itself every milliseconds decided by the user. The application will get the milleseconds value from an environment variable called `ESCRIBA_TIMEOUT`(3000 is the default value).
+The `log-generator` inside `examples` folder will run a Node.js application that will make a request for itself every in an interval defined by the user (in milliseconds). The application will get input value from an environment variable `ESCRIBA_TIMEOUT`(3000 is the default value, this represents 3 seconds)
```
Comment (rctoliveira) [inline README.md]: "The application will get an input value" or "the application will get input values"

File: README.md [hunk 5]
```diff
@@ -96,19 +101,19 @@ app.get('/room/:id', roomController.index)
...
 
-To use log-generator through Docker use this commands inside the log-generator folder:
+To use `log-generator` through Docker use this commands inside the `log-generator` folder:
```
Comment (rctoliveira) [inline README.md]: "use this command" or "use these commands"

**Pull request merged**